django/contrib/admin/checks.py

```diff
@@ -28,6 +29,18 @@ def _issubclass(cls, classinfo):
         return False
 
 
+def _module_exists(dotted_path, paths):
+    """
+    Check if dotted module path is found among paths. Also takes care to
+    ensure any superclass of the queried path is found.
+    """
+    middleware_cls = import_string(dotted_path)
+    for path in paths:
+        if _issubclass(import_string(path), middleware_cls):
```

Comment:
`import_string` can raise `ImportError` which is probably not the best place to be surfaced if you made a typo in your `MIDDLEWARE`.

```suggestion
        try:
            candidate_cls = import_string(candidate_path)
        except ImportError:
            continue
        if _issubclass(candidate_cls, subclass):
            return True
    return False
```